docs/admin_guide.md

```diff
@@ -875,36 +875,59 @@ Test PostgreSQL connections if using a remote server, same with Redis. Ensure no
 Edit your `.env` file:
 
 ```conf
-S3_KEY=
-S3_SECRET=
-S3_BUCKET=media.karab.in
-S3_REGION=eu-central-1
-S3_ENDPOINT=
+S3_KEY=$AWS_ACCESS_KEY_ID
+S3_SECRET=$AWS_SECRET_ACCESS_KEY
+S3_BUCKET=bucket-name
+# safe default for s3 deployments like minio or single zone ceph/radosgw
+S3_REGION=us-east-1
+# set if not using aws s3, note that the scheme is also required
+S3_ENDPOINT=https://endpoint.domain.tld
 S3_VERSION=latest
 ```
 
-And then edit the: `config/packages/oneup_flysystem.yaml` file:
+Then edit the: `config/packages/oneup_flysystem.yaml` file:
 
 ```yaml
 oneup_flysystem:
   adapters:
     default_adapter:
       local:
-        location: "%kernel.project_dir%/public/media"
+        location: "%kernel.project_dir%/public/%uploads_dir_name%"
 
-    kbin.s3_adapter:
+    mbin.s3_adapter:
```

Comment:
I'm fine changing this to `mbin`. Just understand the migration from existing kbin instances might be harder. This WILL break backwards comparability! Ideally support both kbin and mbin for now.

Reply:
oh yes good point, I took care not to rename `%kbin_storage_url%` for this and then nonchalantly rename this one to mbin. thanks for pointing this out, will update soon.

Reply:
I think I'll revert renaming for now, is that alright?

Reply:
To avoid backwards compatibility / regression issues, I think it's safe to revert the name yes. Feel free to create a new PR to support both kbin and mbin.